packages/circuits/crates/libs/safe/src/lib.nr

```diff
@@ -111,7 +111,7 @@ impl<let L: u32> SafeSponge<L> {
     ///
     /// # Arguments
     /// - `input`: Array of field elements to absorb (variable length, must match IO pattern)
-    pub fn absorb(&mut self, input: [Field]) {
+    pub fn absorb(&mut self, input: Vec<Field>) {
```

Comment:
generally speaking, do we wanna default to `Vec<Field>` instead of `[Field]` in most of the code? What are the tradeoffs there? I'm asking since this change in the poly lib in Noir might affect other circuits